sentinel-dashboard/src/main/webapp/resources/app/scripts/controllers/login.js

```diff
@@ -0,0 +1,31 @@
+var app = angular.module('sentinelDashboardApp');
+
+app.controller('LoginCtl', ['$scope', '$state', '$window', 'AuthService',
+    function ($scope, $state, $window, LoginService) {
+        $scope.login = function () {
```

Comment:
Maybe when users jump to the login page, we need to check whether the user has logged in first. If the login session exists and isn't expired, we can directly jump to the index page.

Reply:
Good idea! I'll improve it soon.

Reply:
Besides, I change `sessionStorage` to `localStorage`, in order to keep session in different tab page in browser, it's more convenient for user.